### ISSUE TITLE
Face Recognition Using Difference Component Analysis Implementation

### ISSUE THREAD

**Issue by akshitagupta15june** [akshitagupta15june/Face-X#125]
Difference Component Analysis (DCA) and present its application in face recognition. The DCA removes all the general features of images and compute the difference components of the images. 

Other Algorithms Implementation

https://github.com/akshitagupta15june/Face-X/tree/master/Recognition-Algorithms

**Comment by sagnik1511** [comment]
Hello @akshitagupta15june  di, as I have contributed in your repo will you add me in the readme contributor section?? I'll be very glad if you do so :)

**Comment by akshitagupta15june** [comment]
i think you have already added your name in contributor.md, right?

**Comment by sagnik1511** [comment]
Yes. I told about the reamde file .Can that be updated with my name as a contributor :D ?

**Comment by carrycooldude** [comment]
Hey @akshitagupta15june I want to work on this issue and also if possible can you give some resource for that algo
